@rsvoboda :
```
2026-05-26T07:52:55.0601737Z 07:52:55,041 INFO  mvn: [ERROR] Unresolveable build extension: Plugin io.quarkus.platform:quarkus-maven-plugin:3.33.2 or one of its dependencies could not be resolved:
2026-05-26T07:52:55.0605860Z 07:52:55,041 INFO  mvn: 	io.quarkus.platform:quarkus-maven-plugin:jar:3.33.2 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced
```

It seems, that quarkus MCP switched to platform 3.33.2, which is not released yet

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -30,7 +30,8 @@ jobs:
           git clone -b 3.33 https://github.com/quarkusio/quarkus.git && cd quarkus && ./mvnw -B --no-transfer-progress -s .github/mvn-settings.xml clean install -Dquickly -Dno-test-modules -Prelocations
       - name: Build Quarkus Platform 3.33.999-SNAPSHOT
         run: |
-          git clone -b 3.33 https://github.com/quarkusio/quarkus-platform.git && cd quarkus-platform && ./mvnw versions:set -DnewVersion=3.33.999-SNAPSHOT -DgenerateBackupPoms=false && ./mvnw install -Dquarkus.version=3.33.999-SNAPSHOT -DskipTests -DskipITs
+          # TODO revert commit for excluding the camel-quarkus-integration-test-cxf-soap-grouped
+          git clone -b 3.33 https://github.com/quarkusio/quarkus-platform.git && cd quarkus-platform && ./mvnw versions:set -DnewVersion=3.33.999-SNAPSHOT -DgenerateBackupPoms=false && ./mvnw process-resources -Dquarkus.version=3.33.999-SNAPSHOT -Dquarkus.invoke-platform-project.skip=true && cd generated-platform-project && ../mvnw install -Dquarkus.version=3.33.999-SNAPSHOT -DskipTests -DskipITs -pl "-io.quarkus.platform:camel-quarkus-integration-test-cxf-soap-grouped"
       - name: Tar Maven Repo
         shell: bash
         run: tar -I 'pigz -9' -cf maven-repo.tgz -C ~ .m2/repository

